roles/ceph-mon/tasks/crush_rules.yml

```diff
@@ -12,7 +12,7 @@
     - hostvars[item]['osd_crush_location'] is defined
 
 - name: create configured crush rules
-  command: "{{ docker_exec_cmd }} ceph --cluster {{ cluster }} osd crush rule create-simple {{ item.name }} {{ item.root }} {{ item.type }}"
+  command: "{{ docker_exec_cmd }} ceph --cluster {{ cluster }} osd crush rule create-simple {{ item.name }} {{ item.root }} {{ item.type }} {{ item.device_class | default('') }}"
```

Comment:
This should be `create-replicated`, not `create-simple`. The `create-simple` one does not support device classes.

Reply:
right, I need to rework this